iOS/RCTOrientation/Orientation.m

```diff
@@ -138,11 +138,17 @@ - (void)lockToOrientation:(UIInterfaceOrientation) newOrientation usingMask:(UII
     UIInterfaceOrientation deviceOrientation = _lastDeviceOrientation;
     
     [Orientation setOrientation:mask];
-    UIDevice* currentDevice = [UIDevice currentDevice];
-    
-    [currentDevice setValue:@(UIInterfaceOrientationUnknown) forKey:orientation];
-    [currentDevice setValue:@(newOrientation) forKey:orientation];
-    
+
+    if (@available(iOS 16.0, *)) {
+        NSArray *array = [[[UIApplication sharedApplication] connectedScenes] allObjects];
```

Comment:
![Screen Shot 2022-10-05 at 2 54 55 PM](https://user-images.githubusercontent.com/3821698/194171208-f3bd8403-db2b-4701-927b-12ba31f9c22c.png)

The `connectedScenes` API looks like it's only available in iOS 13 and above. Would something like this work? 

```
      UIWindowScene *scene ;

      if (@available(iOS 13.0, *)) {
        NSArray *connectedScenes = [[[UIApplication sharedApplication] connectedScenes] allObjects];
        if ([connectedScenes count] > 0) {
          scene = (UIWindowScene *)connectedScenes[0];
        }
      } else {
        UIWindow *window = [[UIApplication sharedApplication] keyWindow];
        windowScene = [window windowScene];
      }
```

Reply:
(By the way, I'm not currently a maintainer of `react-native-orientation-locker`, but I'm working on a similar change in https://github.com/discord/react-native-orientation/pull/3 )

Reply:
In my opinion, above code is unnecessary. because `if (@available(iOS 16.0, *))` already check current iOS version is 16 or upper.